knotx-core/knotx-service-adapter/src/main/resources/knotx-service-adapter.json

```diff
@@ -0,0 +1,19 @@
+{
+  "address": "knotx.core.service-adapter",
+  "client.options": {
+    "maxPoolSize": 1000,
+    "keepAlive": false
+  },
+  "services": [
+    {
+      "path": "/service/mock/.*",
+      "domain": "localhost",
+      "port": 3000
+    },
+    {
+      "path": "/service/.*",
+      "domain": "localhost",
+      "port": 8080
+    }
+  ]
+}
```

Comment:
Could we add empty lines at the end of files because it is specified so in *.editorconfig*?